Sources/Host/Darwin/X86_64/MachX86_64.cpp

```diff
@@ -120,7 +120,7 @@ ErrorCode Mach::writeCPUState(ProcessThreadId const &ptid,
 
   mach_msg_type_number_t threadStateCount = x86_THREAD_STATE_COUNT;
   kern_return_t kret = thread_set_state(
-      thread, x86_THREAD_STATE, (thread_state_t)&state, threadStateCount);
+      thread, x86_THREAD_STATE, (thread_state_t)&threadState, threadStateCount);
```

Comment:
Good catch, I've most likely include this issue when fix the name of the structure. Note that the same issue is in readCPUState :D